src/Core.UnitTests/RuleHelpLinkProviderTests.cs

```diff
@@ -30,8 +30,11 @@ public class RuleHelpLinkProviderTests
         [DataRow("javascript:123", "https://rules.sonarsource.com/javascript/RSPEC-123")]
         [DataRow("javascript:SOMETHING", "https://rules.sonarsource.com/javascript/RSPEC-SOMETHING")]
         [DataRow("c:456", "https://rules.sonarsource.com/c/RSPEC-456")]
-        [DataRow("cpp:789", "https://rules.sonarsource.com/cpp/RSPEC-789")]
+        [DataRow("c:NonEmptyCaseWithoutBreak", "https://rules.sonarsource.com/c/RSPEC-NonEmptyCaseWithoutBreak")]
+        [DataRow("cpp:PPIncludeNonStandardCharacters", "https://rules.sonarsource.com/cpp/RSPEC-PPIncludeNonStandardCharacters")]
         [DataRow("php:101112", "https://rules.sonarsource.com/php/RSPEC-101112")]
+        [DataRow("roslyn.sonaranalyzer.security.cs:S2076", "https://rules.sonarsource.com/csharp/RSPEC-2076")]
```

Comment:
Line 36 is the regression test case. I tweak a couple of the existing C/C++ examples to use real legacy rule names.